salt/states/glusterfs.py

```diff
@@ -301,3 +301,83 @@ def add_volume_bricks(name, bricks):
 
     ret['comment'] = 'Adding bricks to volume {0} failed'.format(name)
     return ret
+
+
+def op_version(name, version):
+    '''
+    Add brick(s) to an existing volume
+
+    name
+        Volume name
+
+    version
+        Version to which the cluster.op-version should be set
+
+    .. code-block:: yaml
+
+        myvolume:
+          glusterfs.op_version:
+            - name: volume1
+            - version: 30707
+    '''
+    ret = {'name': name,
+           'changes': {},
+           'comment': '',
+           'result': False}
+
+    current = __salt__['glusterfs.get_op_version'](name)
+
+    if current == version:
+        ret['comment'] = 'Glusterfs cluster.op-version for {0} already set to {1}'.format(name, version)
+        ret['result'] = True
+        return ret
+
+    result = __salt__['glusterfs.set_op_version'](version)
```

Comment:
This state function needs to support `test=true` behavior before actually changing anything. Can you add that, please?